app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/actions/ParseXmlActionImpl.scala

```diff
@@ -32,28 +34,52 @@ class ParseIE815XmlActionImpl @Inject()
 (
   xmlParser: XmlParser,
   cc: ControllerComponents
-)(implicit val executionContext: ExecutionContext) extends BackendController(cc)
+)(implicit val executionContext: ExecutionContext, implicit val emcsUtils: EmcsUtils) extends BackendController(cc)
   with ParseIE815XmlAction
   with Logging {
 
-
   override def refine[A](request: EnrolmentRequest[A]): Future[Either[Result, ParsedXmlRequest[A]]] = {
 
-      request.body match {
-        case body: NodeSeq if body.nonEmpty => parseXml(body, request)
-        case _ =>
-          logger.error("Not valid XML or XML is empty")
-          Future.successful(Left(BadRequest("Not valid XML or XML is empty")))
-      }
+    request.body match {
+      case body: NodeSeq if body.nonEmpty => parseXml(body, request)
+      case _ =>
+        logger.error("Not valid XML or XML is empty")
+        Future.successful(
+          Left(
+            BadRequest(
+              Json.toJson(
+                ErrorResponse(
+                  emcsUtils.getCurrentDateTime,
+                  "XML formatting error", "Not valid XML or XML is empty",
+                  emcsUtils.generateCorrelationId
```

Comment:
Do we really need correlationID here? This may create some confusion for both client and system support. CorrelationId is usually used to follow the path downstream when we call EIS. in this case we do not call EIS but it is just an authentication error. Also the current time I guess is disputable if they need it as the will get this error straight away, so they will now when happened.

Reply:
hmm I guess we can add the correlation id as empty for this. Just wanted to use the same message format as the EIS so we are consistent.